iotedgedev/deploymentmanifest.py

```diff
@@ -190,6 +190,9 @@ def _validate_json_schema(self, schema_object, json_object, schema_type):
                 self.output.warning("%s schema validation failed. Please see previous logs for more details" % schema_type)
             else:
                 self.output.info("%s schema validation passed." % schema_type)
+
+            if error_detected:
```

Comment:
line 189 has if error_detected?

Reply:
besides it already has error_detected, do we still need validation_success?

Reply:
Removed the error_detected and reuse validation_success. We need to declare validation_success outside of try to handle situations with exception.

Reply:
removed error_detected? we have two blocks of error_detected (189 and 194)?

Reply:
Here's the commit that removes error_detected, I removed both of them and use validation_success instead: https://github.com/Azure/iotedgedev/pull/425/commits/60cac6655901b22fad8de6336616c1e28e042373